src/layout/AuthenticatedHeader.tsx

```diff
@@ -0,0 +1,92 @@
+import {FC, memo} from "react";
+import {Link} from "react-router-dom";
+import {Icon} from "@iconify/react";
+import {Button} from "@/components/shadcn-ui/button";
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage,
+} from "@/components/shadcn-ui/avatar";
+import {
+  NavigationMenu,
+  NavigationMenuItem,
+  NavigationMenuLink,
+  NavigationMenuList,
+  navigationMenuTriggerStyle,
+} from "@/components/shadcn-ui/navigation-menu";
+
+export const AuthenticatedHeaderComponent: FC = () => {
```

Comment:
Move this component and AnonymousHeader inside the Navbar component file
DO not export them, only export the Navbar